CHANGELOG.md

```diff
@@ -18,7 +18,6 @@
 - Eslint for code correctness
 - Prettier for formatting
 - leasot for todos
-- Husky & Lint-staged for auto format & lint on commit
```

Comment:
This is a bit misleading, since you're mutating an old changelog entry! It's still true that we added this in 8.0.0. We just removed it now!

Reply:
Right, my bad, just updated this file properly.

In the other hand, I still can't merge, looks like codecov check is mandatory, right?

Reply:
Ah, yeah, it isn't supposed to be required, but it looks like it is. I can merge it.

Reply:
Thank you @serhalp 🙏